src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,277 @@
+                          },
+                      },
+                    ]
+
+Actors:
+  # Clear any pre-existing collection state. This is helpful to avoid errors during local testing.
+  - Name: ClearCollection
+    Type: CrudActor
+    Database: *database
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Threads: 1
+          Collection: *collection
+          Operations:
+            - OperationName: drop
+
+  - Name: CreateTimeSeriesCollection
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *database
+          Operation:
+            OperationMetricsName: CreateTimeSeriesCollection
+            OperationName: RunCommand
+            OperationCommand:
+              { create: *collection, timeseries: { timeField: "time" } }
+
+  - Name: InsertData
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [2]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *database
+          CollectionCount: 1
+          Threads: 1
+          DocumentCount: 1000000
+          BatchSize: 1000
+          Document:
+            _id: { ^Inc: { start: 1 } }
+            # Since step = 900 ms and documentCount = 1,000,000, the first and last document would 
+            # be 900,000,000 ms apart (roughly 10 days). Adding events in this way means we have 
+            # neat buckets because all of our events are in order. However, this is acceptable for 
+            # this workload, because we are testing the speed of bucket unpacking. Inserting 
+            # documents in order allows us to deterministically control how many documents are in 
+            # each bucket, the number of overall buckets, and how many buckets each predicate will # unpack.
+            time: { ^IncDate: { start: "2022-01-01", step: 900 } }
+            # This is the value we are going to query against. We will have ~1013 buckets in this
+            # collection and 988 documents per bucket. By making this an increasing value, we can
+            # write predicates to target certain number of buckets.
+            measurement: { ^Inc: { start: 1 } }
+            # The fields below create larger documents to stress unpacking.
+            payload1: { ^RandomDouble: { min: -10, max: 100 } }
+            payload2: { ^RandomDouble: { min: -10, max: 100 } }
+            payload3: { ^RandomDouble: { min: -10, max: 100 } }
+            payload4: { ^RandomInt: { min: -10, max: 100 } }
+            payload5: { ^RandomInt: { min: -10, max: 100 } }
+            payload6: { ^RandomInt: { min: -10, max: 100 } }
+            payload7: { ^RandomInt: { min: -10, max: 100 } }
+
+  - Name: Quiesce
```

Comment:
added!

Reply:
I thought "Quiesce" forces writes to be flushed to disk and such so that there is no background activity to interfere with the tests. These tests are all read-only, is "Quiesce" still useful?